runtime/src/main/java/org/corfudb/runtime/collections/CorfuRecord.java

```diff
@@ -0,0 +1,14 @@
+package org.corfudb.runtime.collections;
+
+import com.google.protobuf.Message;
+
+import lombok.Data;
+
+/**
+ * Created by zlokhandwala on 2019-08-15.
+ */
+@Data
```

Comment:
Please never use `@data` and `@value` annotations, it's overcomplicated. Use not compound annotations like `@Getter`, `@ToString` and so on

Reply:
Will make the change.